src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,134 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private int size;
+    private final float loadFactor;
+    private Node<K, V>[] table;
+    private int capacity;
+
+    public MyHashMap() {
+        this.capacity = 16;
+        this.loadFactor = 0.75f;
+        this.size = 0;
```

Comment:
you don't need this line, you can remove it because this variable already has this value without additional initialization by default